prog/weaveexec/Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM alpine
+#FROM alpine
+FROM jeanblanchard/alpine-glibc
```

Comment:
That is correct.  The changes in the docker file bind mount in /etc/cni/plugins to get the the executables in there.  And they need to be statically compiled to work.